src/SlackMessages.ts

```diff
@@ -82,8 +80,20 @@ export function atEveryone() {
     return "<!everyone>";
 }
 
+function hasItems(arr: any[]) {
+    return arr != null && arr.length > 0;
```

Comment:
`arr != null` will validate as `arr !== null && arr !== undefined` - you have to add that check or arr.length will result in favorite JS error `Cannot read property 'length' of undefined`